src/java.base/linux/classes/jdk/internal/platform/cgroupv2/CgroupV2Subsystem.java

```diff
@@ -143,7 +141,7 @@ private long getFromCpuMax(int tokenIdx) {
             return CgroupSubsystem.LONG_RETVAL_UNLIMITED;
         }
         // $MAX $PERIOD
-        String[] tokens = cpuMaxRaw.split("\\s+");
+        String[] tokens = cpuMaxRaw.split(" ");
```

Comment:
This seems OK. According to https://docs.kernel.org/admin-guide/cgroup-v2.html#format